lucene/core/src/java/org/apache/lucene/index/IndexWriter.java

```diff
@@ -3179,9 +3317,9 @@ private long prepareCommitInternal() throws IOException {
       SegmentInfos toCommit = null;
       boolean anyChanges = false;
       long seqNo;
-      MergePolicy.MergeSpecification onCommitMerges = null;
-      AtomicBoolean includeInCommit = new AtomicBoolean(true);
-      final long maxCommitMergeWaitMillis = config.getMaxCommitMergeWaitMillis();
+      MergePolicy.MergeSpecification pointInTimeMerges = null;
+      AtomicBoolean hasTimedOut = new AtomicBoolean(false);
```

Comment:
Should we rename this to stopCollectingMergedReaders? I find that name better.

Reply:
++ I change it to a better name